README.md

```diff
@@ -7,4 +7,5 @@ This is a collection of creative and enjoyable GitHub Profile READMEs. Feel free
 - [simonw](https://github.com/simonw) - a self-updating profile using GitHub Actions; featured on [Hacker News](https://news.ycombinator.com/item?id=23807881)
 - [sindresorhus](https://github.com/sindresorhus) - a cheeky loading spinner
 - [terrytangyuan](https://github.com/terrytangyuan) - a collection of badges linking to various platforms
-- [zhiiyang](https://github.com/zhiiiyang) - fetches their latest tweet, renders it to a PNG using headless Chrome (from an R script) and adds it to her profile
\ No newline at end of file
+- [zhiiyang](https://github.com/zhiiiyang) - fetches their latest tweet, renders it to a PNG using headless Chrome (from an R script) and adds it to her profile
+- [tomholford](https://github.com/mpociot) - Spotify now playing + visitor counters. 
```

Comment:
@petrroll Did you mean to use another label here? the link is pointing to `mpociot` but the link text is my own GH name